scripts/style/shellcheck_skip.txt

```diff
@@ -23,9 +23,7 @@ image/templates/common/port-forward.sh
 image/templates/common/setup-central.sh
 image/templates/common/setup-scanner.sh
 image/templates/common/setup.sh
-image/templates/sensor/kubernetes/delete-sensor.sh
```

Comment:
How could you delete this from `shellcheck_skip.txt`?

Reply:
I fixed the warnings